cli.js

```diff
@@ -170,6 +171,56 @@ Usage: $0 <command> [options]` // eslint-disable-line comma-dangle
   }, async (argv) => {
     await handleCommand(argv, listHandler);
   })
+  .command('triggers', 'Manages triggers', yargs0 => yargs0
+      .demandCommand()
+      .command('upsert', 'Creates a new trigger or updates an existing one', (yargs1) => {
+        yargs1
+          .usage('Usage: $0 triggers upsert <trigger name> [options]')
+          .positional('trigger name', {
```

Comment:
do we need "name" here?
in comparison functions print `bn deploy <function>` not `bn deploy <function name>`

Reply:
Changing

Reply:
Not changed

Reply:
leaving it `trigger name` as you agreed (me and @bergundy) that it's the best option.
`name` isn't clear enough
`trigger` alone looks weird